coralproject/cay
# CoralUI Enhancements

## Changelog

This PR adds
-  Support for Loading States in CoralButton
-  Support for sizes and icon sizes
-  Expand API for CoralButton Effects
-  Add CoralDialog Component
-  Add CoralTabBar and CoralTabs Component
-  Enzyme Test Coverage

And fixes issues reported here: https://github.com/coralproject/cay/issues/718
## How do I test this PR?
- `git checkout ui-enhancements` and `npm install`
- Check that everything is working as expected and as [Issue 718](https://github.com/coralproject/cay/issues/718) says.
- Dialogs, tabs, and the rest of the UI should look like the Styleguide looks.
- Run `npm run test`

@coralproject/frontend
